sdb/command.py

```diff
@@ -84,3 +84,25 @@ def call(self,
              objs: Iterable[drgn.Object]) -> Optional[Iterable[drgn.Object]]:
         # pylint: disable=missing-docstring
         raise NotImplementedError
+
+    @classmethod
+    def help(cls, name: str, verbose: bool = False):
```

Comment:
I think that we have to take the `name` here if you want to be able to do `help <alias>` and have it print the help message starting with `alias: `.  So I wouldn't see this as a workaround for the not being able to access `self.name`, which is intentionally not a class-variable.  If you didn't want this behavior, you could start the help message with `cls.name[0]: `.